packages/server/lib/errors.js

```diff
@@ -583,7 +583,7 @@ const getMsgByType = function (type, ...args) {
           The \`component\`.\`devServer\` method must be a function with the following signature:
 
           \`\`\`
-          devServer: (cypressConfig: DevServerConfig, devServerConfig: ComponentDevServerOpts) {
+          devServer: (cypressConfig, devServerConfig) {
```

Comment:
In all of the dev servers, we call the positional arguments to `devServer()`  `cypressDevServerConfig, devServerConfig` but inside cypress we seem to call it `cypressConfig`. Is there any way to update this so that it's consistently `cypressDevServerConfig` everywhere?